package.json

```diff
@@ -174,6 +174,11 @@
           "type": "array",
           "default": [],
           "description": "A list of repositories to query for issues."
+        },
+        "github.host": {
```

Comment:
Would it be difficult to support multiple hosts?

Reply:
I think, if you set multiple hosts, you need to set multiple user names and passwords for each hosts.

Reply:
Sounds good.